pkg/operator/operator.go

```diff
@@ -57,7 +57,7 @@ type defaultReportingOperator struct {
 
 	kubeClient        corev1.CoreV1Interface
```

Comment:
As a side note, this also seems a bit strange to me as to why we would use a `corev1.Corev1Interface` instead of a `kubernetes.Interface`, but this isn't something we need to tackle here.

Reply:
I'm going to start an issue with a list of these :smile:

Reply:
https://github.com/kube-reporting/metering-operator/issues/1357